### ISSUE TITLE
[postgres] Join with array type column

### ISSUE THREAD

**Issue by atonamy** [typeorm/typeorm#5471]
**Issue type:**

[X] question
[ ] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[X] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[X] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

I have two entities

```
@Entity({
  name: 'products'
})
class Product  {
  @PrimaryColumn({
    name: 'id',
    type: 'uuid'
  })
  id: string = uuid()
  @Column({
    name: 'name'
  })
 name: string = ''

@Column({
    name: 'images',
    type: 'array'
  })
 imageIds: string[] = []

images: Image[]
}

@Entity({
  name: 'images'
})
class Image  {
  @PrimaryColumn({
    name: 'id',
    type: 'uuid'
  })
  id: string = uuid()
  @Column({
    name: 'url'
  })
 url: string = ''
}
```

Now I want query something like this:
```
SELECT id, name, i.*
FROM products p
JOIN images i ON i.id = ANY(p.images)
WHERE .... //some condition
```

Basically establish one to many relations using array column in  prodcuts table instead of keep id of product in the image table
How to achieve this with typeorm?

**Comment by zfben** [comment]
I have same question.

**Comment by adlanarifzr** [comment]
Anyone know how to do this?

**Comment by imnotjames** [comment]
This is not currently possible.

**Comment by ianmateusAsaas** [comment]
Is it still not possible to perform this operation?
